Add disclaimer to readme (#32)
pivlab/manugen-ai: README.md

> <div style="clear: both;">&nbsp;</div>

Author: I'm not sure how GitHub flavored markdown handles HTML floats. I'd leave it in to be safe.

**🚧 NOTE**:

> This product was initially made as part of a hackathon under very tight time and resource limitations which resulted in rapidly improvised architecture.

Reviewer: @vincerubinetti what do you think about this suggestion? It's looking ready to be merged.

> Because this product was initially made as part of a hackathon under very tight time and resource limitations, its architecture had to be rapidly improvised, and as such is a bit "rough around the edges".

Reviewer 2: Simplifying a bit. Also removing "rough..." (we can let them judge without our bias 🙂 ).
```suggestion
This product was initially made as part of a hackathon under very tight time and resource limitations which resulted in rapidly improvised architecture.
```

> Our experienced team of engineers intends to turn this brainstorm/proof-of-concept into a fully-fledged, robust system through ongoing funded work.

Reviewer 2: I feel we can accurately say this is a "proof" as it materialized outside our brains.
```suggestion
Our experienced team of engineers intends to turn this proof-of-concept into a fully-fledged, robust system through ongoing funded work.
```
